Add explicit type conversion

minnyres opened the pull request:
To avoid compiling error with clang, explicit type conversion from "size_t" to "int" is required.

persmule commented:
Please write the cast as `(type)expression` rather than `type(expression)`， consistent with other codes around.

minnyres committed:
Add explicit type conversion

To avoid compiling error with clang, explicit type conversion from "size_t" to "int" is required.

minnyres commented:
I have rewritten the cast as  `(type)expression`.

persmule commented:
Okay. Your change is accepted.
